_test/test_utilities/test_get_revert_bytestream.m

```diff
@@ -12,72 +12,71 @@
             this = this@TestCase(name);
         end
         function test_obj_conversion(this)
-            
+
             x=1000;
             bs = get_bytestream_from_obj(x);
             assertTrue(isa(bs(1),'uint8'));
             assertEqual(numel(bs),72);
-            
+
             xc = get_obj_from_bytestream(bs);
-            
+
             assertEqual(x,xc);
             %-----------------
             y = 'abra_cadbra';
             bs = get_bytestream_from_obj(y);
             assertTrue(isa(bs(1),'uint8'));
             assertEqual(numel(bs),80);
-            
+
             yc = get_obj_from_bytestream(bs);
-            
+
             assertEqual(y,yc);
             %-----------------
             z=struct('yyy',y,'xx',x);
-            
+
             bs = get_bytestream_from_obj(z);
             assertTrue(isa(bs(1),'uint8'));
             assertEqual(numel(bs),216);
-            
+
             zc = get_obj_from_bytestream(bs);
-            
+
             assertEqual(z,zc);
             %-----------------
-            
+
             t= IX_fermi_chopper();
             t.radius = 10;
             t.name = 'sloppy';
-            
+
             bs = get_bytestream_from_obj(t);
             assertTrue(isa(bs(1),'uint8'));
             assertEqual(numel(bs),5128);
-            
+
 %            tc = get_obj_from_bytestream(bs);
-            
+
 %            assertEqual(t,tc);
-            
+
         end
-        
+
         function test_mex_nomex(this)
-            if matlab_version_num>8.02 % Matlab 2014a can not currently run mex files build for lower matlab versions
-                return
+            if matlab_version_num>8.02
+                skipTest('Matlab 2014a can not currently run mex files build for lower matlab versions');
```

Comment:
I appreciate that this is an existing test, rather than something you have created, but the test here checks that matlab is 2014a or later (so worth adding "and greater") but does not appear to have a test for the mex file versions.